static/sass/config/_typography.scss

```diff
@@ -0,0 +1,20 @@
+// Typography Config
+// =================
+
+/// Base styles used on page-title H elements 
```

Comment:
well, likely the h1 since it is `page-title` but sometimes the page title might be less important than, say, the article titles on the page... as in, "blog" doesnt do much, but the <article><h1> might be more important ... So this @include title; could be used on the most important h elements instead of limiting it to the h1.